website/server/controllers/api-v3/tasks.js

```diff
@@ -530,6 +530,14 @@ api.scoreTask = {
 
     if (!task) throw new NotFound(res.t('taskNotFound'));
 
+    if (task.completed && (task.type === 'daily' || task.type === 'todo') && direction === 'up') {
+      throw new NotAuthorized('Your session is outdated. Please refresh the page.');
+    }
+
+    if (!task.completed && (task.type === 'daily' || task.type === 'todo') && direction === 'down') {
+      throw new NotAuthorized('Your session is outdated. Please refresh the page.');
```

Comment:
Also, the test changes should probably be in [test/api/v3/integration/tasks/POST-tasks_id_score_direction.test.js](https://github.com/HabitRPG/habitica/blob/develop/test/api/v3/integration/tasks/POST-tasks_id_score_direction.test.js).